megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -2282,8 +2283,7 @@ public void gameEnd(GameEndEvent e) {
 
             // Allow players to save their living units to a file.
             // Don't bother asking if none survived.
```

Comment:
Remove this comment if it's not going to be true anymore

Reply:
Done